r/src/embedding.r

```diff
@@ -0,0 +1,12 @@
+runEmbedding <- function(req, type) {
+    if (type == "pca") {
+        # Leaving this here to add parameters in the future.
+        # RunPCA(data, npcs = 50, features = VariableFeatures(object=data), verbose=FALSE)
+        return(Embeddings(data, reduction = "pca"))
+    } else {
+        # data <- FindNeighbors(data, k.param = 20, annoy.metric = "cosine", verbose=FALSE) #default method
+        # message("Running embedding")
+        # data <- RunUMAP(data, reduction='pca', dims = 1:10, verbose = F, umap.method = "uwot-learn")
+        return(Embeddings(data, reduction = "umap"))
+    }
+}
```

Comment:
Could we also have a version that does tSNE? We kind of need it as the platform currently only allows users to select PCA and tSNE.

Reply:
> Could we also have a version that does tSNE? We kind of need it as the platform currently only allows users to select PCA and tSNE.

Yes, it'll be very straightforward to implement so I can add it to this pull request because either way we are going to look at it tomorrow with Sebastian and Juanlu after we discuss some stuff regarding the ingest script.